nixos/modules/system/activation/bootspec.nix

```diff
@@ -79,9 +75,16 @@ let
 in
 {
   options.boot.bootspec = {
-    enable = lib.mkEnableOption (lib.mdDoc "Enable generation of RFC-0125 bootspec in $system/bootspec, e.g. /run/current-system/bootspec");
+    enable = lib.mkEnableOption "the generation of RFC-0125 bootspec in $system/boot.json, e.g. /run/current-system/boot.json"
+      // { default = true; internal = true; };
+    enableValidation = lib.mkEnableOption (''the validation of bootspec documents for each build.
```

Comment:
```
warning: option boot.bootspec.enableValidation description uses DocBook
```

This also looks to be the case with `boot.bootspec.enable`...?

Reply:
This is fixed now